packages/create-react-native-library/templates/example/example/$package.json

```diff
@@ -6,7 +6,14 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "start": "react-native start"
+    "start": "react-native start",
+    "get:testbutler": "curl -f -o ./test-butler-app.apk https://repo1.maven.org/maven2/com/linkedin/testbutler/test-butler-app/2.2.1/test-butler-app-2.2.1.apk",
+    "detox:ios:build:release": "detox build -c ios.sim.release",
+    "detox:ios:test:release": "detox test -c ios.sim.release",
+    "detox:android:build:release": "detox build -c android.emu.release",
+    "detox:android:test:release": "yarn get:testbutler && detox test --configuration android.emu.release --loglevel trace --headless --take-screenshots failing",
+    "detox:android:build:debug": "detox build -c android.emu.debug",
+    "detox:android:test:debug": "detox test -c android.emu.debug"
```

Comment:
I think it's time to separate the package.json for expo and native projects into 2.